app/Phragile/Burndown.php

```diff
@@ -0,0 +1,107 @@
+<?php
+namespace Phragile;
+
+class Burndown {
+	// TODO: ideally these should come from a cached call to maniphest.querystatuses
+	private static $STATUS_OPEN = ['stalled', 'open'];
+
+	public function __construct(\Sprint $sprint, TaskList $tasks, PhabricatorAPI $phabricator)
+	{
+		$this->sprint = $sprint;
+		$this->tasks = $tasks;
+		$this->phabricator = $phabricator;
+		$this->days = $this->daysInSprint();
+	}
+
+	private function daysInSprint()
+	{
+		$days = [];
+
+		for ($day = strtotime($this->sprint->sprint_start);
+		     $day <= strtotime($this->sprint->sprint_end);
+		     $day += 60*60*24)
+		{
+			$days[] = $day;
+		}
+
+		return $days;
+	}
+
+	public function getDays($format = 'M j')
```

Comment:
This is called both from within this class and from within a template. Slightly odd, not saying this is bad